conda-recipes/main-library/copy_lib.sh

```diff
@@ -4,7 +4,6 @@
 mkdir -p $PREFIX/lib
 mkdir -p $PREFIX/bin
 mkdir -p $PREFIX/include/sls
```

Comment:
We should add the following as well.
cp build/install/bin/sls_detector $PREFIX/bin/.